README.md

```diff
@@ -1,3 +1,164 @@
-# OpenSlides client
+# OpenSlides Client
 
-Webclient for OpenSlides which ...
+Web client for OpenSlides which is written in Angular.
```

Comment:
Hm. The ... which ... clause was ment to describe what this part of OpenSlides is doing. But if "client" is self explaining you may write only "Web client for OpenSlides 4+". Please also adjust repository description in GitHub.